docs/deploy.md

```diff
@@ -548,3 +548,102 @@ This exposes a JSON-RPC interface to the Hardhat Network. Use the address, **htt
+
+Use the following parameters to set connect to Yuma Testnet:
+
+```
+Network name:  Yuma Testnet 
+New RPC URL:  https://yuma-testnet.horizenlabs.io/ethv1 
+Chain ID: 1662 
+Currency symbol: ZEN
+```
+
+In this exercise, the [EON Explorer](https://yuma-explorer.horizen.io/) is connected to the Yuma Testnet. The smart contract was developed and deployed to the Yuma Testnet network. The verification process will be performed using the **Via flattened source code** option in the EON Explorer.
+
+### Flatten the Smart Contract
+
+Before using the EON Explorer, the smart contract you want to verify has been deployed in the Yuma Testnet network. You need to flatten the smart contract if the contract inherits from other contracts or if it uses a library. Flattening a smart contract refers to combining all Solidity code into one file instead of multiple source files, thereby having the imported code be embedded in the same file. 
+
+All smart contract development tools have a source code flattener. For example: 
+
+* **Truffle** - uses the [Truffle Flattener](https://www.npmjs.com/package/truffle-flattener)
+* **Hardhat** - uses the [Hardhat flattener](https://hardhat.org/hardhat-runner/docs/advanced/flattening) 
+* The [POA Solidity Flattener](https://github.com/poanetwork/solidity-flattener) can be used to flatten a smart contract
+* **Remix** - provides a built-in flattener tool
+ 
+**Note:** In the **Remix Explorer**, select your smart contract file ( **.sol** ) to flatten, then right click to the flattener tool. 
+
+### Check for the Deployed Contract
+
+In the [EON Explorer](https://yuma-explorer.horizen.io/), perform the following steps to begin the verification process:
+
+1. Once you have deployed your contract, your developer tool will display a contract address to check the pending transaction. Copy the contract address.
+
+2. Open the EON Explorer and paste the contract address into the **Search** field.
+
+![alt_text](/img/docs/deploy/contract-address.png)
+
+3. The **Contract Address Details** page appears. Review the information displayed. In the lower pane, click **Code**.
+
+![alt_text](/img/docs/deploy/address-details.png)
+
+4. The **Contract Creation Code** page appears. Click **Verify & Publish**. 
+
+![alt_text](/img/docs/deploy/verify-publish.png)
+
+5. The **New Smart Contract Verification** page appears. It displays the contract address and various methods for verifying. By default, the **Via flattened source code** is selected.
+
+![alt_text](/img/docs/deploy/viaflattenedsource.png)
+
+6. In the **New Smart Contract Verification** page, click **Next**. A form page appears. 
+
+![alt_text](/img/docs/deploy/verifyform.png)
+
+Set the form with the following:
+
+  * **Contract name:** The name of the class called in the **.sol** file is the contract name. For example, in contract Owner {.., Owner is the contract name.
```

Comment:
is `Owner {...` missing a closing brace?

Reply:
corrected